lib/ads/server_side_ad_manager.js

```diff
@@ -303,6 +314,12 @@ shaka.ads.ServerSideAdManager = class {
    * @private
    */
   onError_() {
+    const now = Date.now() / 1000;
```

Comment:
Counting time until error as time until ads were loaded is a bit weird to me. What's the rational behind it?

Reply:
Because in this case the loadhas failed but you can continue the loadwith another sequence. It makes sense to me to have the information on this, but if you think it doesn't have it for you, I can remove it.

Reply:
I have finally removed it